atom/browser/api/atom_api_session.cc

```diff
@@ -142,6 +142,16 @@ void SetUserAgentInIO(scoped_refptr<net::URLRequestContextGetter> getter,
           user_agent));
 }
 
+void ResolvePromise(atom::util::Promise promise) {
```

Comment:
Can these helpers be put as static methods on the promise helper to avoid future duplication in our code base?

Reply:
yeah i need to do this

Reply:
Haha, you beat me to it 😄